ui/src/components/song/SongEditorMobileView.tsx

```diff
@@ -0,0 +1,298 @@
+import { FC, ReactElement, useState } from 'react';
```

Comment:
I think a huge chunk of this code is the same as the one in desktop? Can combine it using the useMediaQuery like in Randall's pr? But ima still approve for now lol can do this in future cycles.

Reply:
iya it's the same but just different orders of the form fields 😢